### ISSUE TITLE
Running Tests in parallel using the `--parallel` pest option

### ISSUE THREAD

**Issue by keith-ferney** [pestphp/pest-intellij#487]
When I add the `--parallel` to the `Test Runner Options` field in my Run/Debug Configurations the test run does not behave as expected

it currently does not recognize that the tests are running in parallel, it only shows the command output, not the list of tests and the statuses

**Comment by olivernybroe** [comment]
Duplicate of #375
